module-2/calculating-games-of-chance/your-code/your-code.ipynb

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
- There's something wrong here, the odds results seems to be  same as the probability
```Python
blackjack_o = blackjack_p/(1-blackjack_p)
print("odds of blackjack = %s" %(blackjack_o)) 

twoAs_o = twoAs_p/(1-twoAs_p)
print("odds of  two As = %s" %(twoAs_o))
#odds of blackjack = 0.012213740458015267
#odds of  two As = 0.0015105740181268882
```
- Try to express odds in a fractional way 2:1, 7:1,.... , it's one of the most common ways of expressing odds.